src/main/java/uk/gov/companieshouse/web/accounts/service/transaction/impl/TransactionServiceImpl.java

```diff
@@ -1,18 +1,32 @@
 package uk.gov.companieshouse.web.accounts.service.transaction.impl;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
 import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
 import uk.gov.companieshouse.web.accounts.service.transaction.TransactionService;
 
 @Service
 public class TransactionServiceImpl implements TransactionService {
 
+    @Autowired
+    ApiClientService apiClientService;
```

Comment:
Im guessing this class is a temporary thing until we do the private sdk?

Reply:
No, this service layer will handle transaction api requests via the SDK :)